cmake/Modules/Packages/MLIAP.cmake

```diff
@@ -0,0 +1,29 @@
+# if PYTHON package is included we may also include Python support in MLIAP
+set(MLIAP_ENABLE_PYTHON_DEFAULT OFF)
+if(PKG_PYTHON)
+  find_package(Cythonize)
+  if(Cythonize_FOUND)
+    set(MLIAP_ENABLE_PYTHON_DEFAULT ON)
+  endif()
+endif()
+
+option(MLIAP_ENABLE_PYTHON "Build MLIAP package with Python support" ${MLIAP_ENABLE_PYTHON_DEFAULT})
+
+if(MLIAP_ENABLE_PYTHON)
+  find_package(Cythonize REQUIRED)
+  if(NOT PKG_PYTHON)
+    message(FATAL_ERROR "Must install PYTHON package for MLIAP with Python support")
+  endif()
+
+  set(MLIAP_CYTHON_DIR ${CMAKE_BINARY_DIR}/cython)
+  file(MAKE_DIRECTORY ${MLIAP_CYTHON_DIR})
+  add_custom_command(OUTPUT  ${MLIAP_CYTHON_DIR}/mliap_model_python_couple.cpp ${MLIAP_CYTHON_DIR}/mliap_model_python_couple.h
+            COMMAND            ${CMAKE_COMMAND} -E copy ${LAMMPS_SOURCE_DIR}/MLIAP/mliap_model_python_couple.pyx ${MLIAP_CYTHON_DIR}/mliap_model_python_couple.pyx
+            COMMAND            ${Cythonize_EXECUTABLE} ${MLIAP_CYTHON_DIR}/mliap_model_python_couple.pyx -3
```

Comment:
For readability I would move the `-3` to the front.

Reply:
The -3 shouldn't need to be there, the pyx file has a language_level declaration at the top.

Reply:
> The -3 shouldn't need to be there, the pyx file has a language_level declaration at the top.

i removed most of those. many didn't seem to have an effect, anyway. i consider it a bad idea to have command line flags embedded in source code unless it cannot be avoided. the flag was added to make things more explicit.

Reply:
I had the others around for building this as a standalone module, which is a separate endeavor. I don't have a problem with them removed for lammps:master.

But the cython language_level and distutils : c++ flags are pretty key because they actually give information about how the source file should be read by humans as well, I think it is good to leave them there.

Reply:
...and those two settings had been retained in the source.